src/olympia/search/filters.py

```diff
@@ -459,6 +518,16 @@ def get_es_query(self):
         return [Q('nested', path='colors', query=query.Bool(filter=clauses))]
 
 
+class AddonRatingQueryParam(AddonThresholdQueryParam):
+    query_param = 'ratings'
+    es_field = 'ratings.average'
```

Comment:
Actually I just realized, this field is not indexed, so that wouldn't work properly. You need to change that in the indexer and force a reindex.

Reply:
Alternatively we could search on `bayesian_rating` instead, which is indexed and usually better for search purposes, even though it's not the exact same value users see.

Reply:
it seemed to work locally?

Reply:
mmm if it worked locally then you probably don't have a real mapping set up (it can happen if you trigger the indexing of an add-on before running `reindex`), I would run `reindex --wipe` to ensure you do. It shouldn't be possible in ES to query/filter against non-indexed fields.

Reply:
You should get an Internal Server Error with an Elasticsearch traceback looking like this:
```
failed to create query: {
  "bool" : {
    "filter" : [
      {
        "terms" : {
          "status" : [
            4
          ],
          "boost" : 1.0
        }
      },
      {
        "exists" : {
          "field" : "current_version",
          "boost" : 1.0
        }
      },
      {
        "term" : {
          "is_disabled" : {
            "value" : false,
            "boost" : 1.0
          }
        }
      },
      {
        "range" : {
          "ratings.average" : {
            "from" : 1.0,
            "to" : null,
            "include_lower" : false,
            "include_upper" : true,
            "boost" : 1.0
          }
        }
      }
    ],
    "adjust_pure_negative" : true,
    "boost" : 1.0
  }
}
```

Reply:
https://github.com/mozilla/addons-server/pull/17602/commits/ae527dcb9002a0860538c7cf85f49c02340ca341 adds ratings.average to the ES index